jetty-core/jetty-openid/src/main/config/etc/jetty-openid.xml

```diff
@@ -27,13 +27,13 @@
   </Call>
   <Call name="addBean">
     <Arg>
-      <New id="OpenIdConfiguration" class="org.eclipse.jetty.ee10.security.openid.OpenIdConfiguration">
+      <New id="OpenIdConfiguration" class="org.eclipse.jetty.security.openid.OpenIdConfiguration">
         <Arg name="issuer"><Property name="jetty.openid.provider" deprecated="jetty.openid.openIdProvider"/></Arg>
         <Arg name="authorizationEndpoint"><Property name="jetty.openid.provider.authorizationEndpoint"/></Arg>
         <Arg name="tokenEndpoint"><Property name="jetty.openid.provider.tokenEndpoint"/></Arg>
         <Arg name="clientId"><Property name="jetty.openid.clientId"/></Arg>
         <Arg name="clientSecret"><Property name="jetty.openid.clientSecret"/></Arg>
-        <Arg name="authMethod"><Property name="jetty.openid.authMethod" default="client_secret_post"/></Arg>
+        <Arg name="authenticatorName"><Property name="jetty.openid.authenticatorName" deprecated="jetty.openid.authMethod" default="client_secret_post"/></Arg>
```

Comment:
I don't think this is right.
Elsewhere the `authenticatorName` is `OPENID`, while here is `client_secret_post`.
This should really be the client-side mechanism to use, so it should have a different name.
Perhaps `authenticationMethod` or `clientAuthentication[Method|Mode]`?

Reply:
@lachlan-roberts HELP!!! what is "client_secret_post" about?  Why can the openid authenticator change it's method name?

@sbordet I agree it looks strange.... but it started strange so maybe I open and issue, assign to Lachlan and we move on?